GitCommands/RevisionReader.cs

```diff
@@ -403,34 +401,42 @@ public StringLineReader(in ReadOnlySpan<char> s)
 
                 int startIndex = _index;
                 _index += lineLength + 1;
-                return StringPool.Shared.GetOrAdd(_s.Slice(startIndex, lineLength));
+
+                // Authors etc are limited, use a shared string pool
+                ReadOnlySpan<char> s = _s.Slice(startIndex, lineLength);
+                return useStringPool ? StringPool.Shared.GetOrAdd(s) : s.ToString();
             }
 
-            public (string? subject, string? body, bool hasMultiLineMessage) PeekSubjectBody(bool skipBody)
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
How did you decide this attribute is necessary?

Reply:
This routine can be executed 100K every time the revisions are reloaded, so speed is preferred over code size
I considered removing the internal class StringLineReader to give the compiler some more hints.
I doubt this is measurable though.